### ISSUE TITLE
Add an empty search bar to the Tab Tray, allowing the creation of new tabs

### ISSUE THREAD

**Issue by fellowish** [mozilla-mobile/fenix#12301]
The design flow with the tab tray is a little hard to parse through and really slows down my (and others) browsing. There's one main thing I'd like to suggest to make the tab tray more efficient and speed up browsing.

Currently, it takes 3 taps to create a new tab. First you need to get to the tab tray, then you need to go to home by clicking the '**+*_' button and then click on the search bar. While not the worst sequence it_ _is_

* confusing, especially with the "Home button" for the app being something that looks like it would create a new tab, but instead takes you to the location _where_ you make new tabs.

Instead of the tab tray _covering_ the address bar, have it pull up a search bar instead. 

![tab screen flow](https://user-images.githubusercontent.com/28973978/86558555-4faf4780-bf1f-11ea-8828-9416ff0624c6.png)

It will give very quick access to both creating new tabs, while also making the "Home" page feel more like a tool and less like a pitstop. While the other suggestions would be nice, _my biggest concern here is the ease of access to creating new tabs_. This does just that!

Let me know your thoughts.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-3112)

**Comment by boek** [comment]
Opening a new tab now focuses the search bar. This should satisfy this ticket.
